test/api/v3/integration/groups/POST-groups_id_removeMember.test.js

```diff
@@ -129,6 +131,20 @@ describe('POST /groups/:groupId/removeMember/:memberId', () => {
       expect(invitedUserWithoutInvite.invitations.party).to.be.empty;
     });
 
+    it('removes new messages from a member who is removed', async () => {
+      await partyLeader.post(`/groups/${party._id}/chat`, { message: 'Some message' });
+      await partyLeader.sync();
+      await removedMember.sync();
+      await party.sync();
+      expect(removedMember.newMessages).to.not.be.empty;
```

Comment:
Just to be clear about what's happening, can we assert on `removedMember.newMessages[party._id]` instead?